content/03.results.md

```diff
@@ -133,11 +133,11 @@ The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, includi
 ### Mutational co-occurrence and signatures highlight key oncogenic drivers
 
 The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). 
-We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (25/71 or 35.2% of tumors with a mutation in either gene, OR = 32, q = 8.46e-17), _ATRX_ (14/66 or 21.2%, OR = 20, q = 4.43e-8), _NF1_ (10/71 or 14.1%, OR = 8.62, q = 5.45e-5), and _EGFR_ (6/60 or 10%, OR = 18.2, q = 1.6e-4). 
+We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (25/71 or 35.2% of tumors with a mutation in either gene, OR = 32, CI between 15.3 and 66.7, q = 8.46e-17), _ATRX_ (14/66 or 21.2%, OR = 20, CI between 8.4 and 47.7, q = 4.43e-8), _NF1_ (10/71 or 14.1%, OR = 8.62, CI between 3.7 and 20.2, q = 5.45e-5), and _EGFR_ (6/60 or 10%, OR = 18.2, CI between 5 and 66.5, q = 1.6e-4). 
```

Comment:
I think we can probably eliminate the percentages completely once we have the OR and CI values for coocurrence. This may have been discussed and I missed it or forgot about it, so apologies if I am suggesting something other than what was already decided! 

The other change I made here is just to tighten up the CI presentation, while adding some specificity about the CI interval. 
```suggestion
We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
```

Reply:
@jashapiro - thanks for the feedback. I fixed according to your suggestion!